src/Functions/ascii.cpp

```diff
@@ -75,16 +75,30 @@ using FunctionAscii = FunctionStringOrArrayToT<AsciiImpl, AsciiName, AsciiImpl::
 
 REGISTER_FUNCTION(Ascii)
 {
-    factory.registerFunction<FunctionAscii>(
-        FunctionDocumentation{
-        .description=R"(
-Returns the ASCII code point of the first character of str.  The result type is Int32.
-
-If s is empty, the result is 0. If the first character is not an ASCII character or not part of the Latin-1 Supplement range of UTF-16, the result is undefined)
-        )",
-        .examples{{"ascii", "SELECT ascii('234')", ""}},
-        .category = FunctionDocumentation::Category::String
-        }, FunctionFactory::Case::Insensitive);
+    FunctionDocumentation::Description description = R"(
+Returns the ASCII code point as an `Int32` of the first character of string `s`.
```

Comment:
Returns the ASCII code point of the first character of string `s` as an `Int32`.